### PR TITLE
Update readme file

### PR DESCRIPTION
**Pull request by tuteng**
### Motivation

* Update document for pulsar manager release v0.2.0

### Modifications

* Update document for pulsar manager release v0.2.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,6 +104,28 @@ The Pulsar Manager can be deployed as part of [Pulsar Helm Chart](https://github
...
+wget https://dist.apache.org/repos/dist/release/pulsar/pulsar-manager/apache-pulsar-manager-0.2.0/apache-pulsar-manager-0.2.0-bin.tar.gz
+tar -zxvf apache-pulsar-manager-0.2.0-bin.tar.gz
+cd pulsar-manager
+tar -zxvf pulsar-manager.tar
+cd pulsar-manager
+cp -r ../dist ui
+./bin/pulsar-manager
+```
+
+Note, now you can now access it at the following address: frontend => http://localhost:7750/ui/index.html
```
Comment (Huanli-Meng) [inline README.md:119]:
```suggestion
Now, you can access Pulsar manager at the following address: frontend => http://localhost:7750/ui/index.html
```

File: README.md [hunk 2]
```diff
@@ -113,6 +104,28 @@ The Pulsar Manager can be deployed as part of [Pulsar Helm Chart](https://github
...
+
+[Set the administrator account and password](#access-pulsar-manager)
```
Comment (Huanli-Meng) [inline README.md:121]:
```suggestion
[Set the administrator account and password](#access-pulsar-manager).
```

File: README.md [hunk 3]
```diff
@@ -113,6 +104,28 @@ The Pulsar Manager can be deployed as part of [Pulsar Helm Chart](https://github
...
+
+#### Enable bookkeeper visual manager(optional)
```
Comment (Huanli-Meng) [inline README.md:123]:
```suggestion
#### Enable BookKeeper visual manager(optional)
```

File: README.md [hunk 4]
```diff
@@ -113,6 +104,28 @@ The Pulsar Manager can be deployed as part of [Pulsar Helm Chart](https://github
...
+
+Update the configuration file `pulsar-manager/bkvm.conf`, set `bkvm.enabled` to true
```
Comment (Huanli-Meng) [inline README.md:125]:
```suggestion
Update the configuration file `pulsar-manager/bkvm.conf`, and set `bkvm.enabled` to `true`.
```

File: src/README.md [hunk 5]
```diff
@@ -121,27 +121,71 @@ jwt.broker.secret.key=file:///path/broker-secret.key
 
 ```
 export JWT_TOKEN="your-token"
-docker run -it -p 9527:9527 -e REDIRECT_HOST=http://192.168.55.182 -e REDIRECT_PORT=9527 -e DRIVER_CLASS_NAME=org.postgresql.Driver -e URL='jdbc:postgresql://127.0.0.1:5432/pulsar_manager' -e USERNAME=pulsar -e PASSWORD=pulsar -e LOG_LEVEL=DEBUG -e JWT_TOKEN=$JWT_TOKEN -v $PWD:/data apachepulsar/pulsar-manager:v0.1.0 /bin/sh
+docker run -it -p 9527:9527 -p 7750:7750 -e REDIRECT_HOST=http://localhost -e REDIRECT_PORT=9527 -e DRIVER_CLASS_NAME=org.postgresql.Driver -e URL='jdbc:postgresql://127.0.0.1:5432/pulsar_manager' -e USERNAME=pulsar -e PASSWORD=pulsar -e LOG_LEVEL=DEBUG -e JWT_TOKEN=$JWT_TOKEN -v $PWD:/data apachepulsar/pulsar-manager:v0.2.0 /bin/sh
 ```
 
+* `JWT_TOKEN`: Token of superuser configured for broker, generated by `bin/pulsar tokens create --secret-key` or `bin/pulsar tokens create --private-key` command.
```
Comment (Huanli-Meng) [inline src/README.md:127]: update this part based on comments on the pr in the pulsar repo.

File: src/README.md [hunk 6]
```diff
@@ -121,27 +121,71 @@ jwt.broker.secret.key=file:///path/broker-secret.key
...
+* `$PWD/secret`: The folder where the secret key generated by the `bin/pulsar tokens create-secret-key` command are placed locally
+* `REDIRECT_HOST`: the IP address of the front-end server.
+* `REDIRECT_PORT`: the port of the front-end server.
+* `DRIVER_CLASS_NAME`: the driver class name of PostgreSQL.
+* `URL`: the jdbc url of your PostgreSQL database, example: jdbc:postgresql://127.0.0.1:5432/pulsar_manager. The docker image automatically start a local instance of PostgresSQL database.
+* `USERNAME`: the username of PostgreSQL.
+* `PASSWORD`: the password of PostgreSQL.
+* `LOG_LEVEL`: level of log.
+
+### Enable bookkeeper visual manager
```
Comment (Huanli-Meng) [inline src/README.md:176]:
```suggestion
### Enable BookKeeper visual manager
```

File: src/README.md [hunk 7]
```diff
@@ -121,27 +121,71 @@ jwt.broker.secret.key=file:///path/broker-secret.key
...
+
+Update the field `bkvm.enabled` to `true` for the file [bkvm.conf](https://github.com/apache/pulsar-manager/blob/master/src/main/resources/bkvm.conf).
```
Comment (Huanli-Meng) [inline src/README.md:178]:
```suggestion
Set the `bkvm.enabled` field to `true` in the file [bkvm.conf](https://github.com/apache/pulsar-manager/blob/master/src/main/resources/bkvm.conf).
```

**Review by Huanli-Meng — COMMENTED** [review]
leave my comments. PTAL

**Review by sijie — APPROVED** [review]

File: src/README.md [hunk 8]
```diff
@@ -121,27 +121,71 @@ jwt.broker.secret.key=file:///path/broker-secret.key
...
-```
+export PRIVATE_KEY="file:///pulsar-manager/secret/my-private.key"
+export PUBLIC_KEY="file:///pulsar-manager/secret/my-public.key"
+docker run -it -p 9527:9527 -p 7750:7750 -e REDIRECT_HOST=http://localhost -e REDIRECT_PORT=9527 -e DRIVER_CLASS_NAME=org.postgresql.Driver -e URL='jdbc:postgresql://127.0.0.1:5432/pulsar_manager' -e USERNAME=pulsar -e PASSWORD=pulsar -e LOG_LEVEL=DEBUG -e JWT_TOKEN=$JWT_TOKEN -e PRIVATE_KEY=$PRIVATE_KEY -e PUBLIC_KEY=$PUBLIC_KEY -v $PWD:/data -v $PWD/secret:/pulsar-manager/secret apachepulsar/pulsar-manager:v0.2.0 /bin/sh
+```
+
+* `JWT_TOKEN`: the token of superuser configured for the broker. It is generated by the by `bin/pulsar tokens create --private-key` command.
+* `PRIVATE_KEY`: private key path mounted in container, genrated by `bin/pulsar tokens create-key-pair` command.
```
Comment (Huanli-Meng) [inline src/README.md:146]:
```suggestion
* `PRIVATE_KEY`: private key path mounted in container. It is generated by the `bin/pulsar tokens create-key-pair` command.
```

File: src/README.md [hunk 9]
```diff
@@ -121,27 +121,71 @@ jwt.broker.secret.key=file:///path/broker-secret.key
...
+* `PUBLIC_KEY`: public key path mounted in container, genrated by `bin/pulsar tokens create-key-pair` command.
```
Comment (Huanli-Meng) [inline src/README.md:147]:
```suggestion
* `PUBLIC_KEY`: public key path mounted in container. It is generated by the `bin/pulsar tokens create-key-pair` command.
```

File: src/README.md [hunk 10]
```diff
@@ -121,27 +121,71 @@ jwt.broker.secret.key=file:///path/broker-secret.key
...
+* `$PWD/secret`: the folder where the private key and public key generated by the `bin/pulsar tokens create-key-pair` command are placed locally
+* `REDIRECT_HOST`: the IP address of the front-end server.
+* `REDIRECT_PORT`: the port of the front-end server.
+* `DRIVER_CLASS_NAME`: the driver class name of the PostgreSQL database.
+* `URL`: the JDBC URL of your PostgreSQL database, such as jdbc:postgresql://127.0.0.1:5432/pulsar_manager. The docker image automatically start a local instance of the PostgresSQL database.
```
Comment (Huanli-Meng) [inline src/README.md:152]:
```suggestion
* `URL`: the JDBC URL of your PostgreSQL database, such as jdbc:postgresql://127.0.0.1:5432/pulsar_manager. A local instance of the PostgresSQL database can be automatically started by the docker image.
```

File: src/README.md [hunk 11]
```diff
@@ -121,27 +121,71 @@ jwt.broker.secret.key=file:///path/broker-secret.key
...
+docker run -it -p 9527:9527 -p 7750:7750 -e REDIRECT_HOST=http://localhost -e REDIRECT_PORT=9527 -e DRIVER_CLASS_NAME=org.postgresql.Driver -e URL='jdbc:postgresql://127.0.0.1:5432/pulsar_manager' -e USERNAME=pulsar -e PASSWORD=pulsar -e LOG_LEVEL=DEBUG -e JWT_TOKEN=$JWT_TOKEN -v $PWD:/data apachepulsar/pulsar-manager:v0.2.0 /bin/sh
 ```
 
+* `JWT_TOKEN`: the token of superuser configured for the broker. It is generated by the `bin/pulsar tokens create --secret-key` or `bin/pulsar tokens create --private-key` command.
+* `REDIRECT_HOST`: the IP address of the front-end server.
+* `REDIRECT_PORT`: the port of the front-end server.
+* `DRIVER_CLASS_NAME`: the driver class name of the PostgreSQL database.
+* `URL`: the JDBC URL of your PostgreSQL database, such as jdbc:postgresql://127.0.0.1:5432/pulsar_manager. The docker image automatically start a local instance of the PostgresSQL database.
```
Comment (Huanli-Meng) [inline src/README.md:131]:
```suggestion
* `URL`: the JDBC URL of your PostgreSQL database, such as jdbc:postgresql://127.0.0.1:5432/pulsar_manager. A local instance of the PostgresSQL database can be automatically started by the docker image.
```

File: src/README.md [hunk 12]
```diff
@@ -121,27 +121,71 @@ jwt.broker.secret.key=file:///path/broker-secret.key
...
-docker run -it -p 9527:9527 -e REDIRECT_HOST=http://192.168.55.182 -e REDIRECT_PORT=9527 -e DRIVER_CLASS_NAME=org.postgresql.Driver -e URL='jdbc:postgresql://127.0.0.1:5432/pulsar_manager' -e USERNAME=pulsar -e PASSWORD=pulsar -e LOG_LEVEL=DEBUG -e JWT_TOKEN=$JWT_TOKEN -e PRIVATE_KEY=$PRIVATE_KEY -e PUBLIC_KEY=$PUBLIC_KEY -v $PWD:/data -v $PWD/secret-key-path:/pulsar-manager/secret-key-path apachepulsar/pulsar-manager:v0.1.0 /bin/sh
-```
-
-### Enable Github Login
+export SECRET_KEY="file:///pulsar-manager/secret/my-secret.key"
+docker run -it -p 9527:9527 -p 7750:7750 -e REDIRECT_HOST=http://localhost -e REDIRECT_PORT=9527 -e DRIVER_CLASS_NAME=org.postgresql.Driver -e URL='jdbc:postgresql://127.0.0.1:5432/pulsar_manager' -e USERNAME=pulsar -e PASSWORD=pulsar -e LOG_LEVEL=DEBUG -e JWT_TOKEN=$JWT_TOKEN -e SECRET_KEY=$SECRET_KEY -v $PWD:/data -v $PWD/secret:/pulsar-manager/secret apachepulsar/pulsar-manager:v0.2.0 /bin/sh
+```
+
+* `JWT_TOKEN`: the token of superuser configured for the broker. It is generated by the by `bin/pulsar tokens create --secret-key` command.
+* `SECRET_KEY`: secret key path mounted in container, genrated by `bin/pulsar tokens create-secret-key` command.
```
Comment (Huanli-Meng) [inline src/README.md:166]:
```suggestion
* `SECRET_KEY`: the secret key path mounted in container. It is generated by the `bin/pulsar tokens create-secret-key` command.
```

File: src/README.md [hunk 13]
```diff
@@ -121,27 +121,71 @@ jwt.broker.secret.key=file:///path/broker-secret.key
...
-```
+export PRIVATE_KEY="file:///pulsar-manager/secret/my-private.key"
+export PUBLIC_KEY="file:///pulsar-manager/secret/my-public.key"
+docker run -it -p 9527:9527 -p 7750:7750 -e REDIRECT_HOST=http://localhost -e REDIRECT_PORT=9527 -e DRIVER_CLASS_NAME=org.postgresql.Driver -e URL='jdbc:postgresql://127.0.0.1:5432/pulsar_manager' -e USERNAME=pulsar -e PASSWORD=pulsar -e LOG_LEVEL=DEBUG -e JWT_TOKEN=$JWT_TOKEN -e PRIVATE_KEY=$PRIVATE_KEY -e PUBLIC_KEY=$PUBLIC_KEY -v $PWD:/data -v $PWD/secret:/pulsar-manager/secret apachepulsar/pulsar-manager:v0.2.0 /bin/sh
+```
+
+* `JWT_TOKEN`: the token of superuser configured for the broker. It is generated by the by `bin/pulsar tokens create --private-key` command.
+* `PRIVATE_KEY`: private key path mounted in container, genrated by `bin/pulsar tokens create-key-pair` command.
+* `PUBLIC_KEY`: public key path mounted in container, genrated by `bin/pulsar tokens create-key-pair` command.
+* `$PWD/secret`: the folder where the private key and public key generated by the `bin/pulsar tokens create-key-pair` command are placed locally
```
Comment (Huanli-Meng) [inline src/README.md:148]:
```suggestion
* `$PWD/secret`: the folder where the generated private key and public key are placed locally.
```

File: src/README.md [hunk 14]
```diff
@@ -121,27 +121,71 @@ jwt.broker.secret.key=file:///path/broker-secret.key
...
-```
-
-### Enable Github Login
+export SECRET_KEY="file:///pulsar-manager/secret/my-secret.key"
+docker run -it -p 9527:9527 -p 7750:7750 -e REDIRECT_HOST=http://localhost -e REDIRECT_PORT=9527 -e DRIVER_CLASS_NAME=org.postgresql.Driver -e URL='jdbc:postgresql://127.0.0.1:5432/pulsar_manager' -e USERNAME=pulsar -e PASSWORD=pulsar -e LOG_LEVEL=DEBUG -e JWT_TOKEN=$JWT_TOKEN -e SECRET_KEY=$SECRET_KEY -v $PWD:/data -v $PWD/secret:/pulsar-manager/secret apachepulsar/pulsar-manager:v0.2.0 /bin/sh
+```
+
+* `JWT_TOKEN`: the token of superuser configured for the broker. It is generated by the by `bin/pulsar tokens create --secret-key` command.
+* `SECRET_KEY`: secret key path mounted in container, genrated by `bin/pulsar tokens create-secret-key` command.
+* `$PWD/secret`: the folder where the secret key generated by the `bin/pulsar tokens create-secret-key` command are placed locally
```
Comment (Huanli-Meng) [inline src/README.md:167]:
```suggestion
* `$PWD/secret`: the folder where the generated secret key is placed locally.
```

File: src/README.md [hunk 15]
```diff
@@ -121,27 +121,71 @@ jwt.broker.secret.key=file:///path/broker-secret.key
...
+* `REDIRECT_HOST`: the IP address of the front-end server.
+* `REDIRECT_PORT`: the port of the front-end server.
+* `DRIVER_CLASS_NAME`: the driver class name of the PostgreSQL database.
+* `URL`: the JDBC URL of your PostgreSQL database, such as jdbc:postgresql://127.0.0.1:5432/pulsar_manager. The docker image automatically start a local instance of the PostgresSQL database.
```
Comment (Huanli-Meng) [inline src/README.md:171]:
```suggestion
* `URL`: the JDBC URL of your PostgreSQL database, such as jdbc:postgresql://127.0.0.1:5432/pulsar_manager. A local instance of the PostgresSQL database can be automatically started by the docker image.
```

File: README.md [hunk 16]
```diff
@@ -113,6 +104,28 @@ The Pulsar Manager can be deployed as part of [Pulsar Helm Chart](https://github
...
+wget https://dist.apache.org/repos/dist/release/pulsar/pulsar-manager/apache-pulsar-manager-0.2.0/apache-pulsar-manager-0.2.0-bin.tar.gz
+tar -zxvf apache-pulsar-manager-0.2.0-bin.tar.gz
+cd pulsar-manager
+tar -zxvf pulsar-manager.tar
+cd pulsar-manager
+cp -r ../dist ui
+./bin/pulsar-manager
+```
+
+Now you can now access it at the following address: frontend => http://localhost:7750/ui/index.html.
```
Comment (Huanli-Meng) [inline README.md:119]:
```suggestion
Now, you can access it at the following address: frontend => http://localhost:7750/ui/index.html.
```
it refers to Pulsar Manager? if yes, please use Pulsar Manager here.

File: README.md [hunk 17]
```diff
@@ -113,6 +104,28 @@ The Pulsar Manager can be deployed as part of [Pulsar Helm Chart](https://github
...
+
+[Set the administrator account and password](#access-pulsar-manager).
```
Comment (Huanli-Meng) [inline README.md:121]:
```suggestion
For details, see [Set the administrator account and password](#access-pulsar-manager).
```

**Review by Huanli-Meng — COMMENTED** [review]
leave my comments. PTAL

**Review by Huanli-Meng — APPROVED** [review]
LGTM

**Pull request merged**